reactproject/src/ItemHeading.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import ItemCard from './ItemCard';
+
+export default React.memo(function ItemHeading(props){
+
+    return (
+        <>
+        <h1 key={props.id}> {props.content.visibleName} </h1>
+        {props.content["itemList"].map( key => <ItemCard {...props.itemList[key]} id={key.toString()} key={props.id + key} />)}
```

Comment:
i have named incorrect this file. Name should be itemTypeSection means one section for cold Drinks one for hot Drinks and so one. This will help if we want to add some functionality on that section.